### ISSUE TITLE
Finetune on other datasets

### ISSUE THREAD

**Issue by freddododo** [MzeroMiko/VMamba#63]
Hi !  Thank you very much for this repo ;)

By any chance, do you already have a script that could be used to finetune the models trained on Imagenet1k, to other image classification datasets ? I have seen mmsegmentation was used to finetune the models on COCO in paragraph 4.2, but I was wondering if you have something strictly for classification ? 

Or perhaps it is fine to pass a pretrained checkpoint (--pretrained) in main.py, and reduce the number of epochs/warmup epochs ?

**Comment by sivaji123256** [comment]
@freddododo ,I am also trying to do the same.Have you found the solution for it?
